### ISSUE TITLE
Pass a `vfile` in directly

### ISSUE THREAD

**Issue by wooorm** [sindresorhus/awesome-lint#1]
In [`index.js#L23`](https://github.com/sindresorhus/awesome-lint/blob/14fc116c8ff54fc8a13c4a3b7527eb95fb87d400/index.js#L23), instead of doing this:

```js
return pify(run)(fs.readFileSync(readmeFile, 'utf8'));
```

You can do:

```js
var vfile = require('to-vfile');

// ....

var file = vfile.readSync(readmeFile);
file.quiet = true;
return pify(run)(file);
```

...that will set the proper file-path info (and prevent fatal errors from throwing).

**Comment by sindresorhus** [comment]
Nice! That should definitely be better documented. From the docs I got the impression I could only pass a string.
